planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.xml

```diff
@@ -34,15 +34,15 @@
 
   <node pkg="rostopic" type="rostopic" name="auto_approve_lane_change" args="pub /planning/scenario_planning/lane_driving/lane_change_approval std_msgs/Bool 'data: true' -r 10" />
```

Comment:
`rostopic` is not available in ros2.
I'm not sure if there are any good substitute, but I know we can replace it with `executable` to run it as a command.
```suggestion
  <executable cmd="ros2 topic pub /planning/scenario_planning/lane_driving/lane_change_approval std_msgs/msg/Bool 'data: true' -r 10"/>
```